hawkular-alerts-actions/hawkular-alerts-actions-impl/src/main/java/org/hawkular/alerts/actions/standalone/ServiceNames.java

```diff
@@ -33,20 +35,24 @@
 
     private static String JNDI_ACTIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassActionsServiceImpl";
     private static String JNDI_ACTIONS_STANDALONE = "java:global/hawkular-alerts/CassActionsServiceImpl";
-    private static String JNDI_DEFINITIONS_METRICS =
-            "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
+    private static String JNDI_DEFINITIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
     private static String JNDI_DEFINITIONS_STANDALONE = "java:global/hawkular-alerts/CassDefinitionsServiceImpl";
 
     public enum Service {
-        ACTIONS_SERVICE,
-        DEFINITIONS_SERVICE
+        ACTIONS_SERVICE, DEFINITIONS_SERVICE
     }
 
     private static Map<Service, String> services;
 
     static {
         services = new HashMap<>();
-        if (System.getProperty(HAWKULAR_ALERTS_ACTIONS_ENV, STANDALONE).equals(STANDALONE)) {
+        String env = null;
+        try {
+            env = (String) new InitialContext().lookup("java:comp/env/" + HAWKULAR_ALERTS_ACTIONS_ENV);
```

Comment:
Why the change to move this from system property to env ?
Sorry I don't see it.
For consistency we should use system properties as it is the preferred way.
This has not affect about if it works or not, but I don't think is good to have these differences at this level.